libqtile/widget/crypto_ticker.py

```diff
@@ -74,14 +82,36 @@ def __init__(self, **config):
         if self.symbol == "":
             self.symbol = "$"
 
+        if self.api in self.QUERY_URL_DICT:
+            self.query_url = self.QUERY_URL_DICT[self.api]
+        else:
+            apis = sorted(self.QUERY_URL_DICT.keys())
+            logger.error(
+                "%s is not a valid API. Use one of the list: %s.",
+                self.api,
+                str(apis),
+            )
+
     @property
     def url(self):
-        return self.QUERY_URL.format(self.crypto, self.currency)
+        return self.query_url.format(self.crypto, self.currency)
 
     def parse(self, body):
+        if self.api not in self.QUERY_URL_DICT:
+            return "Invalid API"
+
         variables = dict()
         variables["crypto"] = self.crypto
         variables["symbol"] = self.symbol
-        variables["amount"] = float(body["data"]["amount"])
+        variables["amount"] = self.getAmountFromBodyResponse(body)
 
         return self.format.format(**variables)
+
+    def getAmountFromBodyResponse(self, body):
```

Comment:
Please could you convert this to snake_case, just to be consistent with the code base? thanks

Reply:
The function was removed. Commit: 25096b15e9602a690d0649e79e195f7ecdce0ccc